### ISSUE TITLE
Install on Apple Mac

### ISSUE THREAD

**Issue by carbonmetrics** [factor/factor#3098]
Hi! I have installed factor with:

`brew install --cask factor`

and added this to `.zshrc`:

`export PATH=$APPDIR/factor:"$PATH"`

and confirmed that Rosetta 2 is installed:

`/usr/bin/pgrep -q oahd && echo Yes || echo No`

but Factor does not start from the shell. 

A download from the Factorlang website gives a `dmg` file, nothing is installed when I click on it. It opens a folder, from which you can then double-click `Factor`, which opens the Listener. 

In summary, how do I install Factor on an Apple Mac?

**Comment by mrjbq7** [comment]
Regarding zshrc, i guess make sure you're using a new shell with that location in the environment, and make sure that ``$APPDIR`` is where the factor folder is installed to.

**Comment by carbonmetrics** [comment]
> If you use `brew`, it installs I believe to `/Applications/factor`, so you can run the command line:
> 
> ```
> $ cd /Applications/factor
> $ ./factor
> ```
> 
> Or the graphical interface:
> 
> ```
> $ open /Applications/factor/Factor.app
> ```
> 
> Regarding the `dmg`, it is meant to install like most macOS applications that distribute with dmg's, you can copy the `factor` folder to some location that you want to run it from.

thank you! this worked.

**Comment by mrjbq7** [comment]
Great!
